common/persistence/visibility/visibility_manager_test.go

```diff
@@ -112,23 +134,49 @@ func (s *VisibilityManagerSuite) TestRecordWorkflowExecutionStarted() {
 	s.NoError(s.visibilityManager.RecordWorkflowExecutionStarted(context.Background(), request))
 
 	// no remaining tokens
-	err := s.visibilityManager.RecordWorkflowExecutionStarted(context.Background(), request)
+	err = s.visibilityManager.RecordWorkflowExecutionStarted(context.Background(), request)
 	s.Error(err)
 	s.ErrorIs(err, persistence.ErrPersistenceLimitExceeded)
 }
 
 func (s *VisibilityManagerSuite) TestRecordWorkflowExecutionClosed() {
+	startTime := time.Now().UTC()
+	executionTime := startTime.Add(1 * time.Minute)
+	closeTime := startTime.Add(2 * time.Minute)
 	request := &manager.RecordWorkflowExecutionClosedRequest{
 		VisibilityRequestBase: &manager.VisibilityRequestBase{
 			NamespaceID:      testNamespaceUUID,
 			Namespace:        testNamespace,
 			Execution:        &testWorkflowExecution,
 			WorkflowTypeName: testWorkflowTypeName,
+			StartTime:        startTime,
+			ExecutionTime:    executionTime,
```

Comment:
Is `ExecutionTime` the time the Workflow "started" to be executed by the Worker? Whereas the `StartTime` is when the Workflow was scheduled?